src/components/Charts/RowChart.jsx

```diff
@@ -93,7 +93,8 @@ export class RowChart extends React.Component {
     // deep copy
     // do this to prevent REDUX pollution
     const rows = cloneDeep( data ).filter( o => {
-      if ( o.name ) {
+      if ( o.name && printMode ) {
```

Comment:
> Print view - do not render "visualize trends" text from row chart instead of hiding with CSS to improve spacing

I would check the truth table on this.  It is only true when o.name AND printMode

Reply:
that's intentional.   I have some unit tests that don't have "name" set in the input objects (pretty much just an array of anything)

Normally we want to just render / pass everything to the row chart, and here I want to filter the data.